### PR TITLE
Update Az.Accounts help

### PR DESCRIPTION
**Pull request by msJinLei**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts/help/Get-AzContext.md [hunk 1]
```diff
@@ -93,6 +94,7 @@ The name of the context
 Type: System.String
 Parameter Sets: GetSingleContext
 Aliases:
+Accepted values: AzureSDKTest (0b1f6471-1bf0-4dda-aec3-cb9272f09590) - 54826b22-38d6-4fb2-bad9-b7b93a3e9c5a - leijin@azuresdkteam.onmicrosoft.com
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/Get-AzContext.md:97]: I think This should be removed

File: src/Accounts/Accounts/help/Remove-AzContext.md [hunk 2]
```diff
@@ -90,6 +90,7 @@ The name of the context
 Type: System.String
 Parameter Sets: RemoveByName
 Aliases:
+Accepted values: AzureSDKTest (0b1f6471-1bf0-4dda-aec3-cb9272f09590) - 54826b22-38d6-4fb2-bad9-b7b93a3e9c5a - leijin@azuresdkteam.onmicrosoft.com
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/Remove-AzContext.md:93]: Here too

File: src/Accounts/Accounts/help/Rename-AzContext.md [hunk 3]
```diff
@@ -131,6 +131,7 @@ The name of the context
 Type: System.String
 Parameter Sets: RenameByName
 Aliases:
+Accepted values: AzureSDKTest (0b1f6471-1bf0-4dda-aec3-cb9272f09590) - 54826b22-38d6-4fb2-bad9-b7b93a3e9c5a - leijin@azuresdkteam.onmicrosoft.com
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/Rename-AzContext.md:134]: And here

File: src/Accounts/Accounts/help/Select-AzContext.md [hunk 4]
```diff
@@ -88,6 +89,7 @@ The name of the context
 Type: System.String
 Parameter Sets: SelectByName
 Aliases:
+Accepted values: AzureSDKTest (0b1f6471-1bf0-4dda-aec3-cb9272f09590) - 54826b22-38d6-4fb2-bad9-b7b93a3e9c5a - leijin@azuresdkteam.onmicrosoft.com
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/Select-AzContext.md:92]: And here

File: src/Accounts/Accounts/help/Select-AzContext.md [hunk 5]
```diff
@@ -45,7 +45,8 @@ Target future Azure PowerShell cmdlets at the account, tenant, and subscription
 
 Select a subscription and account to target in Azure PowerShell cmdlets in this powershell session. (autogenerated)
 
-```powershell <!-- Aladdin Generated Example --> 
+```powershell
+<!-- Aladdin Generated Example -->
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/Select-AzContext.md:49]: Example format

File: src/Accounts/Accounts/help/Disable-AzContextAutosave.md [hunk 6]
```diff
@@ -33,7 +33,8 @@ Disable autosave for the current user.
 
 Turn off autosaving Azure credentials in this powershell session. (autogenerated)
 
-```powershell <!-- Aladdin Generated Example -->
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/Disable-AzContextAutosave.md:36]: Example syntax

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/Accounts/Accounts/help/Enable-AzContextAutosave.md [hunk 7]
```diff
@@ -46,7 +46,8 @@ Enable-AzContextAutosave
 Allow the Azure credential, account, and subscription information, to be saved and automatically
 loaded when you open a PowerShell window in this PowerShell session. (autogenerated)
 
-```powershell <!-- Aladdin Generated Example -->
```
[the remark this answers is not in the document]
Comment (msJinLei) [reply src/Accounts/Accounts/help/Enable-AzContextAutosave.md]: > Please use the old syntax

Done

File: src/Accounts/Accounts/help/Remove-AzContext.md (hunk 2 above)
Comment (msJinLei) [reply src/Accounts/Accounts/help/Remove-AzContext.md]: > Here too

Done

**Review by isra-fel — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]

**Review by erich-wang — APPROVED** [review]

**Pull request merged**